### PR TITLE
ZKUI-426 // Polishing ui

### PR DESCRIPTION
**Pull request by hervedombya**

### CODE REVIEW

File: src/react/ui-elements/Veeam/VeeamSkipModal.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  exitAction: () => void;
+  modalContent: JSX.Element;
+  title?: string;
+};
+
+const ModalContent = styled.div`
+  max-width: 30rem;
+`;
+
+export const VeeamSkipModal = ({
```
Comment (JBWatenbergScality) [inline src/react/ui-elements/Veeam/VeeamSkipModal.tsx:17]: nit: Maybe we should add a test to make sure this confirm modal is opened when clicking on `Skip Use case configuration `

Comment (hervedombya) [reply src/react/ui-elements/Veeam/VeeamSkipModal.tsx:17]: Done in f00d1b3fc0b1d110e9937196d6dd4f490010f362

**Review by JBWatenbergScality — APPROVED** [review]
LGTM !

**Pull request merged**